x/auth/types/account.go

```diff
@@ -188,6 +189,33 @@ func (acc *BaseAccount) SpendableCoins(_ time.Time) sdk.Coins {
 	return acc.GetCoins()
 }
 
+// MarshalYAML returns the YAML representation of an account.
+func (acc BaseAccount) MarshalYAML() (interface{}, error) {
+	var pubkey string
+	if acc.PubKey != nil {
+		pubkey = sdk.MustBech32ifyAccPub(acc.PubKey)
```

Comment:
This function returns an error so let's not call `Must*` but rather return an error if this fails.

Reply:
ACKd